parallel/spmv/run_spmv.jl

```diff
@@ -0,0 +1,122 @@
+using Base: nothing_sentinel
+#!/usr/bin/env julia
+if abspath(PROGRAM_FILE) == @__FILE__
+    using Pkg
+    Pkg.activate(@__DIR__)
+    Pkg.instantiate()
+end
+include("../../deps/diagnostics.jl")
+print_diagnostics()
+
+using MatrixDepot
+using BenchmarkTools
+using ArgParse
+using DataStructures
+using JSON
+
+# Parsing Arguments
+s = ArgParseSettings("Run Parallel SpMV Experiments.")
+@add_arg_table! s begin
+    "--output", "-o"
+    arg_type = String
+    help = "output file path"
+    "--dataset", "-d"
+    arg_type = String
+    help = "dataset keyword"
+    "--method", "-m"
+    arg_type = String
+    help = "method keyword"
+    "--accuracy-check", "-a"
+    action = :store_true
+    help = "check method accuracy"
+end
+parsed_args = parse_args(ARGS, s)
+
+# Mapping from dataset types to datasets
+datasets = Dict(
+    "uniform" => [
+        OrderedDict("size" => 1000, "sparsity" => 0.1),
+        OrderedDict("size" => 10000, "sparsity" => 0.1),
+    ],
+    "FEMLAB" => [
+        "FEMLAB/poisson3Da",
+        "FEMLAB/poisson3Db",
+    ],
+)
+
+# Mapping from method keywords to methods
+include("serialize_default_implementation.jl")
```

Comment:
a small naming thing: the word "serialize" means something different than "serial"